integration-tests/security/src/main/resources/application.properties

```diff
@@ -1,19 +1,21 @@
 # Authentication properties
 org.acme.openapi.security.auth.basic_auth/username=john
 org.acme.openapi.security.auth.basic_auth/password=test
+customAuth.additionalApiTypeAnnotations=@org.eclipse.microprofile.rest.client.annotation.RegisterClientHeaders,@org.eclipse.microprofile.rest.client.annotation.RegisterProvider(io.quarkiverse.openapi.generator.it.security.auth.DummyApiKeyAuthenticationProvider.class)
+quarkus.openapi-generator.codegen.enable-security-generation=false
 
 # since the file name has a space, we use the URI representation of this space here to not break the properties file
 # see the RFC3986 for more info https://datatracker.ietf.org/doc/html/rfc3986
 quarkus.openapi-generator.codegen.spec.open_weather_yaml.base-package=org.acme.openapi.weather.v2
+quarkus.openapi-generator.codegen.spec.open_weather_yaml.enable-security-generation=true
```

Comment:
Isn't it default? It's better to remove it, so we ensure that the default is working.